module.yaml

```diff
@@ -1,4 +1,4 @@
-name: "gitops-db2-instance"
+name: gitops-cp-db2-oltp
```

Comment:
It looks like this only listed the default dependencies, but in your test/stages directory I see a lot more files (cp4d operator, cp4d instance, etc...).  If those are in-fact dependencies (which I believe they are), then we need all of those dependencies listed in `module.yaml`.  

It also looks like there's still a file in test/stages for `[stage2-db2u-operator.tf](https://github.com/cloud-native-toolkit/terraform-gitops-cp-db2-oltp/blob/init-version/test/stages/stage2-db2u-operator.tf)`.   The contents of which are commented out.  I don't think that is needed anymore.

Reply:
Readme comments updated

Reply:
I have added all the dependencies in module.yaml. I understand db2u operator is one of the dependencies to install db2oltp Subscription and Db2oltpService instance, Please advice?